.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
```

Comment:
We might want to start sharing the common config accross centreon & centreon-react-components. I'm not sure what's the best way to go about it. Put it in centreon-react-components and consume it in centreon?

Reply:
one approach for ESLint would be to create a config plugin: https://eslint.org/docs/developer-guide/shareable-configs